plugins/rapid7_insightappsec/plugin.spec.yaml

```
@@ -4,7 +4,8 @@ products: [insightconnect]
 name: rapid7_insightappsec
 title: Rapid7 InsightAppSec
 description: This plugin allows for the creation, configuration, and starting of scans. The plugin can also retrieve scan results and logging related to the execution of the scan
-version: 1.0.3
+version: 2.0.0
+supported_versions: ["Rapid7 InsightAppSec"]
```

Same as above, add timestamp

---

Same as above

---

If there is no timestamp to report testing the plugin, please simply leave the `supported_versions` field empty.
```suggestion
supported_versions: []
```